girder_volview/utils.py

```diff
@@ -76,26 +77,38 @@
 
 
 def isSessionItem(item):
-    if item and SESSION_ZIP_EXTENSION in item["name"]:
-        return True
-    return False
+    return item and SESSION_ZIP_EXTENSION in item["name"]
 
 
 def isSessionFile(file):
-    name = file.get("name")
-    if name.endswith(SESSION_ZIP_EXTENSION):
-        return True
-    return False
+    return file.get("name").endswith(SESSION_ZIP_EXTENSION)
+
+
+def isTiffFile(file):
+    return (
+        file.get("name").endswith((".tif", ".tiff"))
+        or file.get("mimeType") == "image/tiff"
+    )
+
+
+def isDicomFile(file):
+    return (
+        file.get("name").endswith(".dcm") or file.get("mimeType") == "application/dicom"
+    )
 
 
 def isLoadableFile(file):
-    name = file.get("name")
-    knownExtension = name.endswith(LOADABLE_EXTENSIONS)
-    if knownExtension:
+    if isTiffFile(file) or isDicomFile(file):
+        item = Item().load(file.get("itemId"), level=AccessType.READ)
+        if isTiffFile(file):
+            return not item.get("largeImage")
+        if item.get("meta", {}).get("dicom", {}).get("Modality", "") is "SM":
```

Comment:
You probably what this to be `==` not `is`.

Reply:
Fixed.